### DISCUSSION TITLE
I lost all my coin, i dont understand any, please help me

### DISCUSSION THREAD

**Discussion by khangtroc** [Chia-Network/chia-blockchain#5949] [Support]
Hello bothers
today i open my wallet and it show 0 xch, i was farmed 12 xch this month, where are my coin gone ? how can i get it back ? pls help me, how i can contact support team ? i very sad now
my first address: xch1qulu7j09s8n77tv2wv8rf9e90k34rd5q7wvh5uwcj2zavfmcsxxsyrk6mk
the second : xch12y66t88lymp8kp0atrcfqrv754dulhrapstf6efaqt2hajjaqkgq68k9g3
Chia explorer show they spent at height 312052, same with my second address , i very confuse :((
![1](https://user-images.githubusercontent.com/84728824/119373032-2ee01400-bce2-11eb-994b-6e60e30fed8d.jpg)
![2](https://user-images.githubusercontent.com/84728824/119373036-30a9d780-bce2-11eb-9fd7-cbf82147b793.jpg)
![3](https://user-images.githubusercontent.com/84728824/119373039-31426e00-bce2-11eb-91a4-7071e4900473.jpg)
![4](https://user-images.githubusercontent.com/84728824/119373043-31db0480-bce2-11eb-9dd5-efb9ea27b154.jpg)
![5](https://user-images.githubusercontent.com/84728824/119373047-32739b00-bce2-11eb-8254-e2e605aae93d.jpg)

**Comment by pluto2021** [comment]
Might be too late now, but you might want to manage two addresses on two different machines on completely irrelevant networks. V1.1.6 seems to sync ok over cellular network without the 8444 port forwarding setting on a router (for cellular network, you cannot set that at all). I could not find any info on working with two wallet addresses on a same node. You could not even get them synced at a same time. Probably something that need to be fixed by the dev team.

**Reply by khangtroc** [reply]
i dont want to manage anything, just a one click mouse on new address button, just a click by me when i newbie setup, if that click make me lost coin so why the dev team make that button :(((

**Reply by lvcivs** [reply]
Having more than one wallet receive address is not a problem at all. So if you have 5 friends you can click the "new address" button 5 times, tell each friend one of these addresses and when they send you money all of it will still end up in your one wallet. So no need for 2 machines, unless you want to have 2 separate wallets with 2 separate sets of keys.

**Reply by pluto2021** [reply]
Problem of having 5 wallet addresses on a same machine is that you can only sync one wallet at a time. If you clicked to create a new wallet address, you have to sync that wallet from zero height (OMG) !!! Having 5? You have to sync them one after the other, while you sync the other 4, the 1st wallet would stop where it were. This is a PITA. Any solution to this problem?  If the software is buggy handling more than one wallet address on a same machine, it could be buggy sending XCH to each other within those wallet. If your wallet is not in sync to the height you made the transactions, you certainly will not see the amount showing up at least, if you are lucky not losing your reward.

**Reply by lvcivs** [reply]
You misunderstand. A normal user will have only **one** wallet but can have **multiple** wallet addresses, all pointing to the same wallet. It is like having 5 mailboxes in front of your house through which mail is delivered to you, but all mail ultimately ends up on your desk.

**Reply by room101-dev** [reply]
> i dont want to manage anything, just a one click mouse on new address button, just a click by me when i newbie setup, if that click make me lost coin so why the dev team make that button :(((

The GUI doesn't work, nothing on it works, your first step is 'never use the gui', regards to dev, they don't give a hell about the user.
CHIA is designed to make it easy to steal, anybody can ssh into the the chia-node-daemon, and dump your private keys & meumonic, then its 'gone' as they say.

The only thing that works is to mine on HPOOL and keep your funds on their server, at least if they get hacked have 90% they can mend the block-chain, you can't.

like somebody above says',  have another offline system for just the wallet, and use that to sweep funds, into a offline private-key;

The problem is that unless you have infinite time, and are a computer expert you can't fix these problems, and chia-dev doesn't give a 'feces' about anything, other than their IPO.

What I do, is do all my mining on HPOOL and keep my funds far away from the 'chia sofware' which is 100% feces

**Comment by lvcivs** [comment]
Did you make any transactions on 21. May? If so, where did you send to?

I see 3 outgoing transactions from your first wallet address on that day: https://www.chiastatus.com/addresses/xch1qulu7j09s8n77tv2wv8rf9e90k34rd5q7wvh5uwcj2zavfmcsxxsyrk6mk 

It looks like 0.25 XCH and 11 XCH left your wallet in block 312'052 and then another 1 XCH in block 312'056.

https://www.chiastatus.com/addresses/xch12y66t88lymp8kp0atrcfqrv754dulhrapstf6efaqt2hajjaqkgq68k9g3
Then I see that you received 0.25 XCH on your second address in block 312'052.

Did you perhaps send money to your own second wallet receive address?

**Reply by khangtroc** [reply]
i did not anything just power on my some pc and plot and farm, even i have not other wallet in exchange. If i send or spent the i dont say lost coin, noone torch my pc,  was my pc hacked ? help me pls

**Reply by lvcivs** [reply]
May I ask if you ever input your 24 words anywhere, for example in a pool software? 

Did you recently scan your PC for malware? 

Did you by any chance download the Windows installer on 10th May through the wiki? (There was an incident with malware being linked to from the wiki on that day).

**Reply by khangtroc** [reply]
no bro, i did not anything, did not join any pool , my pc run smooth 3 or 4 year ago, just auto update windows from microsoft

**Reply by lvcivs** [reply]
Disregard the previous comment. The OP has only one wallet with two receive addresses. Clearly not a syncing issue.

**Reply by pluto2021** [reply]
When you click "Create a new private key", would it create a 2nd wallet? not just a 2nd receiving address? If so, how to create a 2nd address using a same wallet?

**Reply by khangtroc** [reply]
I click button new address, it created 2nd address, i have one 24 keys and one wallet, chiastatus show 4 days ago when my 2nd received 0.2499 xch, at the same time all my 12 xch in 1st address gone

**Reply by lvcivs** [reply]
> When you click "Create a new private key", would it create a 2nd wallet? 

Yes.

> not just a 2nd receiving address? If so, how to create a 2nd address using a same wallet?

Once you have selected your keys, go to the tab "Wallet" and you will see a button "New Address" (third image from the top in the post above).

**Reply by pluto2021** [reply]
Never paid attention. Good to know. Thanks man.

**Comment by khangtroc** [comment]
i format my pc, re install windows, re install chia app, re type 24 my keys and syn from 0. When my wallet syn to about at height 28-30k , it still show my 12 XCH and show 1st address in receive address box,  but when syn finished it show 0 XCH and auto show 2nd address in receive address box. Is it bug ? this bug make me lost my 12xch ?

**Reply by lvpcguru** [reply]
There are a lot of these same issues being posted starting a few hours ago. There may be a bug the other option is that someone may have completed a script that scans for open RPC ports as that may let them tell your open wallet to start a transaction.

the ports that should NEVER be accessible from the outside:
    Daemon: 55400
    Full Node: 8555
    Farmer: 8559
    Harvester: 8560
    Wallet: 9256

If someone forwards all ports to their node and not just the 8444 one needed to sync this could easily be open to the internet. check https://www.yougetsignal.com/tools/open-ports/ to make sure those are not open.

example of an api call [https://github.com/Chia-Network/chia-blockchain/wiki/RPC-Interfaces](https://github.com/Chia-Network/chia-blockchain/wiki/RPC-Interfaces)

**Reply by khangtroc** [reply]
i just checked, my router pfsense no forward these port, only 8444

**Reply by lvcivs** [reply]
Those outgoing transactions are on the blockchain, so reinstalling / re-syncing won't help. As soon as it catches up to the present, the wallet balance will show 0 XCH again.

**Reply by pluto2021** [reply]
Since all transactions are logged permanently on the chain, could the receiving address be easily found out  for the missing rewards? or if not, it is lost due to a bug?

**Reply by lvpcguru** [reply]
Its easy to find the public address of it yes. One of the ones I tryed to track today started bouncing and splitting like it was being mixed to obscure it.

 buuuut this is crypto, the entire point is that its irreversible and not easily linked to the end parties.

**Reply by pluto2021** [reply]
So some angry solo farmer started to steal the rewards? Any offline wallet we can use to secure any humble rewards we have so far?

**Comment by rkitvn** [comment]
https://www.chiastatus.com/addresses/xch1qulu7j09s8n77tv2wv8rf9e90k34rd5q7wvh5uwcj2zavfmcsxxsyrk6mk

check this .. i see transfer 12 xch

**Reply by khangtroc** [reply]
I see too, but i can not understand why my xch have these transferred. I was been hacked or bug or dev removed my xch ?

**Reply by room101-dev** [reply]
> https://www.chiastatus.com/addresses/xch1qulu7j09s8n77tv2wv8rf9e90k34rd5q7wvh5uwcj2zavfmcsxxsyrk6mk

https://www.chiaexplorer.com/blockchain/address/xch1qulu7j09s8n77tv2wv8rf9e90k34rd5q7wvh5uwcj2zavfmcsxxsyrk6mk

I think there are more 'backdoors' than we think we know of, the fact is the blockchain wasn't ready for prime-time

Trouble is we need to 'sign' the plots to make them, so we need to keep the keys around;;

The only solution I see  is to mine on hpool, keep your xch away from the 'blockchain', wait until there are solid 3rd party wallets that can be trusted 'offline', or just transfer your earned xch from hpool to btc or ust right away and be done, or move your chia to a trusted exchange.

It's rather been clear all along here that its super easy to get into a chia node, and get the private-key, as the RPC hands it out on a silver plate with a sign that says 'steal me', I'm going to guess that the the crook has a server that scrapes, and at this point just sweeps high-value, most of us don't keep large amounts around.

As these thefts become more common the tools get published, you can expect them to go after the <1 xch stuff

The problem here is that the CHIA-DEV morons, with their GUI, put everything into one place, the wallet, node,server,famer, plots, ... so once the hacker gets in, there free to do as they wish. Then there is the issue of these dozens of 'plotman' variants, how doesn't body know if any if this crap isn't the vector? I mean once you run a 3rd party tool of any kind from the 'chia santa', they issue a 'chia keys show --dump-mnemomic' in the python script, and send that home, your DONE.

**Reply by XIII39** [reply]
My solution so far has been to keep a separate wallet on a separate network. 

I have multiple internet connections, VPN and hardware firewalls. A dedicated connection on for my farmer, harvesters and plotters. My farmer sends the rewards to a second wallet address which I occasionally sync on a computer that is on a completely different network..

If someone manages to get into my farm, the wallet is 0 always. Sure they might be able to find the wallet address of where I send my rewards but there are no keys and no access though the network to a computer that would contain those keys.
It's messy and honestly there are things that need to be fixed.

Losing 12 Chia is no joke. I feel for you.

If you're using multiple machines, be sure not to install your keys on all the machines, use SSL certs. If you can, send your rewards to a wallet that is synced on a separate network.. If all else fails, send your rewards directly into an exchange.. it's not ideal but for now, at least they would be responsible for any loss.

**Reply by room101-dev** [reply]
Probably a good idea  to get somebody to write up a security report on removing the wallet all together, where not needed.

The entire system was designed by a clown, nobody keeps all the keys to the kingdom in one place unlocked. NObody.

I agree, if you can remove the wallet files, and just use  the appropriate ssl/ca cert keys then do it, and leave nothing for the bot-hackers to steal, or sweep

There should be glue in the 'config.yaml' to disable 'chia keys'.

**Comment by tiny0little** [comment]
interested to see the outcome of this ...
clearly see that 12xch left, by design only person with access to your `private key` / `mnemonic seed` is able to to do that ... not developers

**Reply by room101-dev** [reply]
The developers are the one that put the GIANT HOLE into chia and made this possible, **nobody allows the priv-keys & mneumonic to be dumped without a password and 2fa, your covering up for blatant criminal incompetence is mind boggling.**

'chia keys show --dump-mnemomic' 

I work with dozens of blockchains, and nobody on earth does this, probably in test, somebody put this into the code because they were too lazy to enter 12 magic words, just dump & cut&paste.
You say 'the developers' don't have the key, what they heck are you saying, any client on the CPU that issues ''chia keys show --dump-mnemomic' " gets the complete crown-jewels of the kingdom.

ANYBODY can via the network get into chia and issue ''chia keys show --dump-mnemomic' " and then have your keys, that includes the developer. Why do you even bother posting support here, you clearly have not idea of how chia work, or the existence of documented cmds for dumping keys to the public.

Once they have your priv-key, they can sweep your account.
Compeche?????

**Comment by khangtroc** [comment]
Why my last reward is 0.2499 while all my previous rewards were 0.25+1.75 ?
Why when I received last reward all my coin gone too at same time ?
Why i reinstall os and app chia, after synced my receive address always show 2nd address while it show my first address during syncing ?
If i was haked why the hacker only take my xch when i have some eth in my computer still safe ?
And why my some rig farming eth in my LAN always safe a year ago, never been changed wallet address in bat file
Why i scan my pc by some software they said my computer not attacked and i still lost coin.


why the xch wallet has not password or 2fa security?

**Reply by room101-dev** [reply]
> Why my last reward is 0.2499 while all my previous rewards were 0.25+1.75 ?
> Why when I received last reward all my coin gone too at same time ?
> Why i reinstall os and app chia, after synced my receive address always show 2nd address while it show my first address during syncing ?
> If i was haked why the hacker only take my xch when i have some eth in my computer still safe ?
> And why my some rig farming eth in my LAN always safe a year ago, never been changed wallet address in bat file
> Why i scan my pc by some software they said my computer not attacked and i still lost coin.
> 
> why the xch wallet has not password or 2fa security?

NO wallet every does, all crypto works this way, once they have your private key, your money is theres.

'chia keys show' dumps all, and there is even a swtich to dump the mneumonic for those criminals who fear numbers, IMHO chia was designed to enable theft.

Your last comment is telling, it means this is your first crypto. Its been apparent all along that chia, is designed to attrack  noobs; There should be a big banner that says', come plot, and destroy your drives an give your proceeds to charity. It could well be that this 'backdoor' was put in place by the chinese owners of chia. I mean stealing from the west isn't even stealing its like scams are inherently only targeted to the un-informed.

Well so now you know, don't let more than 0.25 XCH accumulate anywhere near the chia blockchain environment. Try to keep all your xch on addresses that have no association with any of your public nodes, miners, or farmers.

**Reply by SiberiaR** [reply]
Did you have a login password on your operating system?

**Reply by room101-dev** [reply]
> Did you have a login password on your operating system?

They track these high value addresses by IP, remember the scam of 100mojo free if you provide an address, then they know the high-value address, they know the IP, then they have a node connect to node, issue a rpc, get the private-key. done.

**Reply by grasuu** [reply]
Did you use the malicious sync script mentioned here?
-> https://www.reddit.com/r/chia/comments/nl94th/psa_potential_security_breach_for_end_users_who/

**Reply by khangtroc** [reply]
I plot and farm only by app chia

**Reply by iromath** [reply]
Did you using the same plot to mine on pool and mine solo?

**Reply by khangtroc** [reply]
I mine solo, never join pool

**Reply by rcamarda390** [reply]
If one suspected their 24 keywords compromised, what is the work around if you have xch?
can I move my plots to a new wallet, as example?

**Comment by hajes** [comment]
"Hpool" is your problem. If you give up your private keys, why the hell, are you people so smart and keep rewards in the same wallet. Then whine about "stolen" coins. You won't leave your keys in your car, either.

**Comment by BrandtH22** [comment]
closing as no new comments, questions, or info
